tensorflow/core/platform/posix/posix_file_system.cc

```diff
@@ -276,4 +277,46 @@ Status PosixFileSystem::RenameFile(const string& src, const string& target) {
   return result;
 }
 
+Status PosixFileSystem::CopyFile(const string& src, const string& target) {
+#if defined(PLATFORM_WINDOWS)
+  return FileSystem::CopyFile(src, target);
+#else
+  string translated_src = TranslateName(src);
+  struct stat sbuf;
+  if (stat(translated_src.c_str(), &sbuf) != 0) {
+    return IOError(src, errno);
+  }
+  int src_fd = open64(translated_src.c_str(), O_RDONLY);
+  if (src_fd < 0) {
+    return IOError(src, errno);
+  }
+  string translated_target = TranslateName(target);
+  int target_fd =
+      open64(translated_target.c_str(), O_WRONLY | O_CREAT,
```

Comment:
Add a comment here explaining all the flags turned on.

Reply:
Done. Thanks.